### DISCUSSION TITLE
Discord Player not working with FFmpeg

### DISCUSSION THREAD

**Discussion by AustinFRG** [Androz2091/discord-player#631] [General]
Hey, I keep getting this error and my music isn't working.
[Discord Player] FFmpeg/Avconv not found! Install via "npm install ffmpeg-static" or download from https://ffmpeg.org/download.html

I have FFmpeg installed; I've tried npm i ffmpeg-static, npm i ffmpeg-static -g, etc.

**Comment by Skelly-debug** [comment]
I get the same error

**Comment by AustinFRG** [comment]
Yeah, I still haven’t gotten mine working
